console-subscriber/src/consequences.rs

```diff
@@ -0,0 +1,105 @@
+use console_api as proto;
+use tracing_causality as causality;
+use tracing_subscriber::Registry;
+
+fn into_proto_span(span: causality::Span) -> proto::consequences::Span {
```

Comment:
if you import all the proto stuff but alias them to prefix `proto`, like `protoSpan`, it could look a bit cleaner